integration_test/third_party_apps_data/applications/rabbitmq/debian_ubuntu/install

```diff
@@ -20,6 +20,10 @@ case $VERSION_ID in
   20.04)
     echo "deb http://ppa.launchpad.net/rabbitmq/rabbitmq-erlang/ubuntu focal main" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
     ;;
+  23.04)
+    # rabbitmq is not supported for Ubuntu 23.04 - install 22.04 instead
```

Comment:
Our customers aren't going to do that, are they? Should we just disable RabbitMQ support on Ubuntu 23.04 instead?